Firebase/Messaging/FIRMessagingRemoteNotificationsProxy.m

```diff
@@ -433,16 +433,16 @@ - (void)application:(GULApplication *)application
  * In order to make FCM SDK compile and compatible with iOS SDKs before iOS 10, hide the
  * parameter types from the swizzling implementation.
  */
-void FCM_swizzle_willPresentNotificationWithHandler(
-    id self, SEL _cmd, id center, id notification, void (^handler)(NSUInteger)) {
+void FCMSwizzleWillPresentNotificationWithHandler(
```

Comment:
Make static since the implementation is not needed to be visible from other object files.